app/controllers/bets_controller.rb

```diff
@@ -0,0 +1,84 @@
+class BetsController < ApplicationController
```

Comment:
After scoping it inside Football matches please chain authorization.

example:
``` authorize [:profile, :public_details, Project] ``